src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -299,6 +333,18 @@ export const PaymentRoute: FC<PaymentRouteProps> = props => {
     } catch (error) {
       handlePaymentError(error)
       setIsSavingPayment(false)
+
+      trackEvent({
+        action: ActionType.errorMessageViewed,
+        context_owner_type: OwnerType.ordersPayment,
+        context_owner_id: props.order.internalID,
+        title: "An error occurred",
+        message:
+          "Something went wrong. Please try again or contact order@artsy.net",
+        error_code: null,
+        flow: "user sets wire transfer as payment method",
+      })
+
```

Comment:
suggestion for future: to prevent duplication and make future updates easier, we could wrap the call to trackEvent in a function with static key/value pairs (action, owner type and id) and call that function with the dynamic values (title, message, etc.).

Reply:
oh it'd have been even better if we could share that function among the routes. 

_I think_ there's no direct way to that though, since each route is loaded separately. perhaps @damassi knows a way.